.changeset/purple-dots-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes styles of `client:only` components not persisting during view transitions
```

Comment:
```suggestion
Fixes styles of `client:only` components not persisting during view transitions in dev mode
```
Ooooh, and reading your PR description more closely, you even bolded this, so probably something for here, too!

It continues to be an amusing pattern to me of writing the good stuff in the PR description itself, which only reviewers will read.  :smile:  The PR title itself starts with `fix` to let other devs know what kind of change this PR is, and so it makes perfect sense that `Fixes...` would similarly be a great start for the entry in the changelog that users will read!